README.md

```diff
@@ -1486,6 +1487,56 @@ let query = Storefront.buildQuery { $0
 }
 ```
 
+#### Customer Update [⤴](#table-of-contents)
+
+Input objects, like `Storefront.MailingAddressInput`, use `Input<T>` type to represent optional fields and distinguish `nil` values from `undefined` values.
```

Comment:
I'm not sure about the phrasing at "Input objects, like `Storefront.MailingAddressInput`, **use** `Input<T>` **type** to represent optional fields..." -- does that work for describing `Input<T>` OR should it be something like "...use **the** `Input<T>` type to represent..." (or "...use `Input<T>` **values** to represent..."?)

Reply:
Well, `Input` is a type that represents the values contained in `Storefront.MailingAddressInput`. I'm almost thinking something like:

> Input objects, like `Storefront.MailingAddressInput`, use values of type `Input<T>` (where `T` is the type of value) to represent optional fields and distinguish `nil` values from `undefined` values (eg. `phone: Input<String>`)

What do you think?

Reply:
@dbart01 does this work?

> Input objects, like `Storefront.MailingAddressInput`, use `Input<T>` (where `T` is the type of value) to represent optional fields and distinguish `nil` values from `undefined` values (eg. `phone: Input<String>`).

Reply:
Perfect 👍